.secrets.env.example

```diff
@@ -41,3 +41,9 @@ GOERLI_FIREBLOCKS_MINTER=<Address (in FB)>
 GOERLI_FIREBLOCKS_RESCUER=<Address (in FB)>
 GOERLI_FIREBLOCKS_BURNER=<Address (in FB)>
 
+#below is all test details for deploying to GAIX network 
+EDGE_DEPLOYER_PRIVATE_KEY=<random wallet used to deploy>
+EDGE_DEVELOPER_PUBLIC_KEY=<random wallet used to deploy>
```

Comment:
what is this used for?

Reply:
Deploying the contract, we are not using any multsig or fb, just a hotwallet.

Reply:
removed it.